tests/models/unets/test_models_unet_2d_condition.py

```diff
@@ -358,7 +358,7 @@ class UNet2DConditionModelTests(ModelTesterMixin, UNetTesterMixin, unittest.Test
     model_class = UNet2DConditionModel
     main_input_name = "sample"
     # We override the items here because the unet under consideration is small.
-    model_split_percents = [0.5, 0.3, 0.4]
+    model_split_percents = [0.5, 0.34, 0.4]
 
```

Comment:
since `UpBlock2D` is not split now, need update 0.3->0,34 split percent to make UT pass